### DISCUSSION TITLE
Create or update file contents doesn't trigger GHA

### DISCUSSION THREAD

**Discussion by sorenmat** [community/community#8255] [Pull Requests] [answered]
Hi,
It seems like using this API https://docs.github.com/en/rest/reference/repos#create-or-update-file-contents to create a file
doesn't trigger a push event to GHA

**Comment by Katsute** [comment, marked as answer]
The `${{ secrets.GITHUB_TOKEN }}` is a special token that doesn't trigger action events, this is to prevent an infinite loop where an action could push a file retriggering the workflow, causing it to push again and repeating over and over again.

If you want to trigger the push event in a workflow you need to use a personal access token.
